common/modules/gcp-project/main.tf

```diff
@@ -503,6 +515,9 @@ locals {
   # Project owners will be empty list if var.project_owner is empty string ""
   project_owners = "${compact(list(var.project_owner))}"
 
+  # Groups that can see the logs in staging.
```

Comment:
This may be not only about logs, any other perms that devs may want on stg.

Reply:
https://issues.gpii.net/browse/GPII-4331 according with the description of the issue it is the only permission needed. I would close this as it is, and reopen in the case that anything else is needed.